bin/node/runtime/src/lib.rs

```diff
@@ -1497,6 +1498,10 @@ impl pallet_uniques::Config for Runtime {
 	type Locker = ();
 }
 
+parameter_types! {
+	pub FeatureFlags: SystemFeatures = SystemFeatures::empty();
```

Comment:
if someone asks me what SystemFeatures are I would say pallet features so why don't we rename `SystemFeatures` to `PalletFeatures` ?

Reply:
Agree! Renamed ;)